area/area.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	fmt.Println("Enter three triangle sides:")
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	var s = (a + b + c) / 2
```

Comment:
Instead of `var <variable_name> = <expression>`, please use short assignment: `<variable_name> :=  <expression>`.

In this case, it should be `s := (a + b + c) / 2`.